<ui version="4.0" >
 <author>Bart Cerneels</author>
 <class>PodcastSettingsBase</class>
 <widget class="QWidget" name="PodcastSettingsBase" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>329</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="MinimumExpanding" hsizetype="MinimumExpanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Podcast Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <layout class="QHBoxLayout" name="m_urlLabelLayout" >
       <item>
        <widget class="KUrlLabel" name="m_feedLabel" >
         <property name="textInteractionFlags" >
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
         <property name="url" >
          <string>Feed URL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_leftClickLabel" >
         <property name="font" >
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="layoutDirection" >
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text" >
          <string>(left click to copy to clipboard)</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLabel" name="textLabel1" >
         <property name="text" >
          <string>Save location:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KUrlRequester" name="m_saveLocation" />
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="m_autoFetchCheck" >
       <property name="toolTip" >
        <string>When checked, Amarok will automatically scan the podcast for updates</string>
       </property>
       <property name="whatsThis" >
        <string>When checked, Amarok will automatically scan the podcast for updates</string>
       </property>
       <property name="text" >
        <string>Automatically scan for updates</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KButtonGroup" name="kbuttongroup" >
       <property name="title" >
        <string>Media Download</string>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QRadioButton" name="m_downloadRadio" >
          <property name="toolTip" >
           <string>Download media as soon as it becomes available</string>
          </property>
          <property name="whatsThis" >
           <string>Download media as soon as it becomes available</string>
          </property>
          <property name="text" >
           <string>Download when a&amp;vailable</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="m_streamRadio" >
          <property name="toolTip" >
           <string>Media must be explicitly downloaded, otherwise the podcast will be played from the remote server.</string>
          </property>
          <property name="whatsThis" >
           <string>Media must be explicitly downloaded, otherwise the podcast will be played from the remote server.</string>
          </property>
          <property name="text" >
           <string>Stream or download on re&amp;quest</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QCheckBox" name="m_purgeCheck" >
         <property name="toolTip" >
          <string>If checked, Amarok will throw away old podcast episodes</string>
         </property>
         <property name="whatsThis" >
          <string>If checked, Amarok will throw away old podcast episodes</string>
         </property>
         <property name="text" >
          <string>Limit &amp;number of episodes</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="spacer16" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>108</width>
           <height>29</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="m_purgeCountLabel" >
         <property name="text" >
          <string>Keep maximum of:</string>
         </property>
         <property name="wordWrap" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="m_purgeCountSpinBox" >
         <property name="toolTip" >
          <string>The maximum number of podcast items to store</string>
         </property>
         <property name="whatsThis" >
          <string>The maximum number of podcast items to store</string>
         </property>
         <property name="suffix" >
          <string> Items</string>
         </property>
         <property name="minimum" >
          <number>1</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="spacer17" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>496</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>KButtonGroup</class>
   <extends>QGroupBox</extends>
   <header>kbuttongroup.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlLabel</class>
   <extends>QLabel</extends>
   <header>kurllabel.h</header>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >kurlrequester.h</include>
  <include location="local" >klineedit.h</include>
  <include location="local" >kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_purgeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_purgeCountSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_purgeCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_purgeCountLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
